.eslintrc

```diff
@@ -9,5 +9,6 @@
   "rules": {
     "no-mixed-operators": [2, { "allowSamePrecedence": true }],
     "jsx-a11y/no-static-element-interactions": 1, // TODO: enable
+    "react/no-did-mount-set-state": 0, // necessary for server-rendering
```

Comment:
Every single one of airbnb's hypernova components works like that already. We can definitely strive to avoid it, but it's not possible to avoid for client-required functionality.